mod_reforged/hooks/config/strings.nut

```diff
@@ -841,7 +848,7 @@ local vanillaDescriptions = [
 				Type = ::UPD.EffectType.Passive,
 				Description = [
 					"Once per battle, upon receiving a killing blow, survive instead with a few hitpoints left and have all damage over time effects (e.g. bleeding, poisoned) cured.",
```

Comment:
change hitpoints to `[Hitpoints|Concept.Hitpoints]` and hyperlink both bleeding and poisoned to their respective effects e.g. `[bleeding+Skill+bleeding_effect], [poisoned|Skill+spider_poison_effect]`